test/models/site_test.rb

```diff
@@ -2,6 +2,8 @@
 
 class SiteTest < ActiveSupport::TestCase
   setup do
+    @default_locale = I18n.locale
+    I18n.locale = :en
```

Comment:
I'll use `ja` for the following points:

* The product is almost used by Japanese.
* If we use `ja`, we can test against localization configuration of error messages.